pkgs/development/python-modules/better-exceptions/default.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+}:
+
+buildPythonPackage rec {
+  pname = "better-exceptions";
+  version = "0.3.3";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `build-system = [ setuptools ]`